### ISSUE TITLE
Poor performance

### ISSUE THREAD

**Issue by llbbcc** [DAI-Lab/RivaGAN#15]
* Python version: 3.9
* Operating System: Linux

### Description

```
from rivagan import RivaGAN

model = RivaGAN()
model.fit('./data/hollywood2', epochs=300, batch_size=12, use_critic=True, use_adversary=True)
model.save('./checkpoints/model.pt')
```
When I use this script for training, I get a few warnings as follows,
```
[mp3float @ 0xe49f1240] Header missing
[mp3float @ 0xe49f1240] Header missing
[mp3float @ 0xe4c01e00] Header missing
```
I don't know if this affects the model. 
My model accuracy has been stuck at about 54%.

What I use is Hollywood2 dataset and it has been processed for successful loading.

Has anyone else experienced the same problem, or has anyone reproduced the results of the paper.

**Comment by langyo** [comment]
It's audio data and wouldn't affect the result.

**Comment by llbbcc** [comment]
> It's audio data and wouldn't affect the result.

Thanks for your reply! The reason I'm having this problem is that Hollywood2 is too small for training a 64bit length model. Moments In Time is great.

**Comment by langyo** [comment]
> > It's audio data and wouldn't affect the result.
> 
> Thanks for your reply! The reason I'm having this problem is that Hollywood2 is too small for training a 64bit length model. Moments In Time is great.

If you need, here are some pre-trained onnx models inside the demo: https://github.com/ShieldMnt/invisible-watermark/tree/main/imwatermark
